### PR TITLE
Hol

### PR DESCRIPTION
**Pull request by WSPluta**
can you review/approve this PR

### CODE REVIEW

**Review by ttscoff — APPROVED** [review]

**Comment by jasperan** [comment]
@WSPluta I think this PR is also having issues with the license before it can be merged

**Comment by ttscoff** [comment]
Looks like one of the fonts has a proprietary license. I can force merge this if needed.

**Comment by WSPluta** [comment]
thanks

**Pull request merged**
